src/goto-instrument/code_contracts.cpp

```diff
@@ -513,20 +513,8 @@ void code_contractst::instrument_call_statement(
 
   exprt called_assigns =
     to_code_with_contract_type(called_symbol.type).assigns();
-  if(called_assigns.is_nil()) // Called function has no assigns clause
-  {
-    // Create a false assertion, so the analysis
-    // will fail if this function is called.
-    goto_programt failing_assertion;
-    failing_assertion.add(goto_programt::make_assertion(
-      false_exprt(), instruction_iterator->source_location));
-    program.insert_before_swap(instruction_iterator, failing_assertion);
-    ++instruction_iterator;
-
-    return;
-  }
-    else // Called function has assigns clause
-    {
+  if(!called_assigns.is_nil()) // Called function has assigns clause
+  {
```

Comment:
The alignment of the old code now inside this conditional is off by two spaces. I don't know if this is something clang-format dislikes, but it would be nice to fix.

Reply:
Implemented as suggested.